src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js

```diff
@@ -133,7 +133,7 @@ setTimeout(async function () {
     let applyFailed = false;
     deltas.forEach(d => {
       try {
-        window.Blazor._internal.applyHotReload(d.moduleId, d.metadataDelta, d.ilDelta)
+        window.Blazor._internal.applyHotReload(d.moduleId, d.metadataDelta, d.ilDelta, d.pdbDelta)
```

Comment:
Was this a miss from the previous PR?

Reply:
The previous PR was in AspNetCore that exposed JSInterop APIs to accept the pdb-delta. This change here modifies the script that is injected in to a page and runs in responds to WebSocket messages from VS / watch.